packages/app/src/runner/ResizablePanels.cy.tsx

```diff
@@ -163,9 +163,10 @@ describe('<ResizablePanels />', { viewportWidth: 1500, defaultCommandTimeout: 40
 
       cy.contains('panel2').should('not.be.visible')
       assertWidth('panel1', defaultPanel1Width)
-      dragHandleToClientX('panel1', 1000)
-      assertWidth('panel1', 1000)
-      assertWidth('panel3', 500)
+      dragHandleToClientX('panel1', 950)
+      dragHandleToClientX('panel1', 955)
```

Comment:
@marktnoonan I was surprised that this test started failing due to panel 1 no longer resizing to 1000px. But it turns out we still account for panel 2's width when deciding panel 1's maximum size, even when panel 2 is hidden. It doesn't look like we disable panel 2 anywhere but the tests, but I'm curious if this behavior is intentional.

Reply:
Agreed on the additional tests, I'll log something to track this. We'll want get a bunch of workflows snapshotted once our Percy workflows are more stable.

As far as the `showPanel2` prop goes, do we even want to continue supporting it? Or do we anticipate a need in the near future?